extensions/common/xwalk_extension_server.cc

```diff
@@ -360,10 +360,9 @@ void XWalkExtensionServer::Invalidate() {
 
 namespace {
 base::FilePath::StringType GetNativeLibraryPattern() {
-  const base::string16 library_pattern = base::GetNativeLibraryName(
-      base::UTF8ToUTF16("*"));
+  base::string16 library_pattern = base::ASCIIToUTF16(base::GetNativeLibraryName("*"));
```

Comment:
base::GetNativeLibraryName() already returns as UTF16 string so don't understand why you need to convert from ASCII to UTF16 here.